packages/plugin-protocol-http/src/utils.spec.ts

```diff
@@ -55,4 +58,8 @@ describe('plugins/http/utils', () => {
         abort(result);
         expect(requestAbort).toHaveBeenCalled();
     });
+    it('get cache flags', () => {
```

Comment:
Would you mind to explain me the purpose of this test?

Reply:
Hm, just to increase test coverage :)

Reply:
If so, then I suggest to remove it)

Reply:
Done